components/freertos/port/port_common.c

```diff
@@ -138,6 +139,11 @@ static void main_task(void* args)
     }
 #endif
 
+// Todo: this could be initialized sooner, but not sure where exactly...
```

Comment:
`xPortStartScheduler()` would probably be a better place to call `stack_mirror_enable()`